node/src/opt.rs

```diff
@@ -163,6 +163,13 @@ pub struct Opt {
     pub expensive_queries_filename: String,
     #[clap(long, help = "Enable debug logging")]
     pub debug: bool,
+    #[clap(
+        long,
+        value_name = "JSON_LOGGING",
+        env = "JSON_LOGGING",
+        help = "output logs in json format"
+    )]
+    pub json_logging: bool,
```

Comment:
This should also be a configurable option. Could have the values: `default` and `json` for now, others can be added later.